scripts/utils/git

```diff
@@ -22,36 +22,42 @@ install_hooks() {
   for hook in $1
   do
     # Check if hook is executable
-    if [ -x scripts/${hook} ]
+    if [! -x scripts/${hook} ]
```

Comment:
but then spaces are mandatory around an expr so it should be 
`if [ ! -x scripts/${hook} ]` or it throws `[!: command not found`